cmds/vpdbootmanager/addentry/add.go

```diff
@@ -0,0 +1,45 @@
+package addentry
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/systemboot/systemboot/pkg/vpd"
+)
+
+const usage = `Usage:
+boot add [netboot [dhcpv6|dhcpv4] [MAC] | localboot [grub|path [Device GUID] [Kernel Path]]]
+
+Use -h for flag options
+`
+
+// Cmd executes the command line parsing for the add command
+func Cmd() {
+	if len(os.Args) < 3 {
+		fmt.Println(usage)
+		os.Exit(1)
+	}
+	switch os.Args[2] {
+	case "netboot":
+		addNetbootEntry()
+	case "localboot":
+
+	}
+}
+
+func addBootEntry(data []byte) error {
+	for i := 1; i < 10; i++ {
```

Comment:
why 10? For how inefficient it may sound, we should iterate over the whole boot variables space, i.e. [0, 9999]. This value should go to a package-level variable, in `pkg/vpdbootmanager`